proto/ibc/applications/transfer/v1/query.proto

```diff
@@ -35,6 +35,11 @@ service Query {
   rpc EscrowAddress(QueryEscrowAddressRequest) returns (QueryEscrowAddressResponse) {
     option (google.api.http).get = "/ibc/apps/transfer/v1/channels/{channel_id}/ports/{port_id}/escrow_address";
   }
+
+  // TotalEscrowForDenom returns the total amount of tokens in escrow based on the denom.
+  rpc TotalEscrowForDenom(QueryTotalEscrowForDenomRequest) returns (QueryTotalEscrowForDenomResponse) {
+    option (google.api.http).get = "/ibc/apps/transfer/v1/total_escrow/{denom}";
```

Comment:
Once upon a time I remember being told that the constant before a user provided argument needed to be plural, so in this case `total_escrows/{denom}`, this is why it is `channels`, `ports`. I was never sure why this would be the case, but does anyone have any idea if this has changed?

Reply:
Yes, the plural naming is the [convention for resources in REST apis](https://restfulapi.net/resource-naming/). But in this case, it would be more appropriate to make it than `denoms/{denom}/total_escrow`. I will change it to that.

Reply:
Turns out that it seems to work with a URI like `denoms/{denom}/total_escrow`!